LockboxXCUITests/LockboxXCUITests.swift

```diff
@@ -6,6 +6,7 @@ import XCTest
 
 let emailTestAccountLogins = "test-b62feb2ed6@restmail.net"
 let passwordTestAccountLogins = "FRCuQaPm"
+let testingURL = "https://wopr.norad.org/~sarentz/fxios/testpages/password.html"
```

Comment:
I don't think this is used anywhere

Reply:
Yes, you are right! forgot to remove

Reply:
@isabelrios Did you want `testingURL` still or shall we yank that before we merge this? I'm ready whenever you're ready. 👍